assets/src/components/appStateWrapper.tsx

```diff
@@ -4,19 +4,31 @@ import useSocket from "../hooks/useSocket"
 import useTimepoints from "../hooks/useTimepoints"
 import useVehicles from "../hooks/useVehicles"
 import DispatchProvider from "../providers/dispatchProvider"
-import { Route, TimepointsByRouteId } from "../skate"
+import { Route, TimepointsByRouteId, UserToken } from "../skate"
 import { initialState, reducer } from "../state"
 import App from "./app"
 
+export const readUserToken = (): UserToken | undefined => {
+  const dataEl = document.getElementById("app")
```

Comment:
Should this be in a `useEffect`? Will this necessarily exist when we run this code? I think of reading from global state as a side effect, but it still might be safe to do without the hook.

Reply:
This will necessarily exist when our app runs, this isn't part of the React app, it's the div the app is written in to.